app/controllers/TransferAgreementController.scala

```diff
@@ -49,6 +49,7 @@ class TransferAgreementController @Inject()(val controllerComponents: SecurityCo
       val transferAgreementStatus: String = consignmentStatus.flatMap(_.transferAgreement).getOrElse("")
       if (transferAgreementStatus == "Completed") {
         Ok(views.html.transferAgreementAlreadyCompleted(consignmentId, transferAgreementForm, options))
+          .withHeaders("Cache-Control" -> "no-store, must-revalidate")
```

Comment:
Could you add some more detail to the commit message for this change? Then someone who doesn't know why the cache-control header is needed will be able to work it out from the commit history.

Reply:
I've removed the new TA page (essentially merged it with the original) as you'll see in my newer commits; should I still update this commit?

Reply:
This has made me think - there are a few pages which use this header, but I don't think the meaning is obvious if you're not familiar with cache controls.

What about having an extension method called something like `.uncached` which is used by every page that needs it? Then you could explain the purpose in a code comment on the method itself.

Give me a shout on Slack if that doesn't make sense, or if you want to talk about the idea.